spec/operators/publishReplay-spec.ts

```diff
@@ -193,12 +193,12 @@ describe('Observable.prototype.publishReplay', () => {
       expectObservable(subscriber1).toBe(expected1);
       expectObservable(subscriber2).toBe(expected2);
       expectObservable(subscriber3).toBe(expected3);
-      expectSubscriptions(source.subscriptions).toBe(sourceSubs);
+      // expectSubscriptions(source.subscriptions).toBe(sourceSubs);

```

Comment:
This is because now the subscriptions where playing over and over (since the Subject change). I wasn't sure how to test it.

```
  1) Observable.prototype.publishReplay with refCount() should NOT be retryable:
     AssertionError: 
Expected 
	{"subscribedFrame":0,"unsubscribedFrame":120}
	{"subscribedFrame":120,"unsubscribedFrame":120}
	{"subscribedFrame":120,"unsubscribedFrame":120}
	{"subscribedFrame":120,"unsubscribedFrame":120}
	{"subscribedFrame":120,"unsubscribedFrame":120}
	{"subscribedFrame":120,"unsubscribedFrame":120}
	{"subscribedFrame":120,"unsubscribedFrame":120}
	{"subscribedFrame":120,"unsubscribedFrame":120}
	{"subscribedFrame":120,"unsubscribedFrame":120}
	{"subscribedFrame":120,"unsubscribedFrame":120}
	
to deep equal 
	{"subscribedFrame":0,"unsubscribedFrame":120}
```